docs/maintainer/infrastructure.md

```diff
@@ -469,6 +469,159 @@ uploaded from the new feedstock, by design.
 Once this is done correctly and the package is uploaded,
 you can then request the conda-forge core devs to archive the old feedstock.
 
+## How package building interacts with infrastructure
+
+Packages in conda-forge are always built through CI.
```

Comment:
```suggestion
Packages in conda-forge are always built through CI.
```

This deserves a footnote or info admonition mentioning [CFEP-03](https://github.com/conda-forge/cfep/blob/main/cfep-03.md).

Reply:
Good call. I added a footnote.